static-metric/Cargo.toml

```diff
@@ -23,7 +23,7 @@ quote = "0.5"
 lazy_static = "1.0"
 
 [dev-dependencies]
-prometheus = "0.4"
+prometheus = "^0"
```

Comment:
why ^0 here?

Reply:
I worry that how the cargo show the version for this crate?

Reply:
Hmm.. It will show "prometheus ^0" like this: https://crates.io/crates/prometheus-static-metric (Already published :smirk: )

Reply:
Anyway, this is a dev dependency, which will not relate to the user.